docs/content/migration/v2-to-v3.md

```diff
@@ -10,28 +10,335 @@ How to Migrate from Traefik v2 to Traefik v3.
 
 The version 3 of Traefik introduces a number of breaking changes,
 which require one to update their configuration when they migrate from v2 to v3.
-The goal of this page is to recapitulate all of these changes, and in particular to give examples,
-feature by feature, of how the configuration looked like in v2, and how it now looks like in v3.
+The goal of this page is to recapitulate all of these changes,
+and in particular to give examples, feature by feature, 
+of how the configuration looked like in v2,
+and how it now looks like in v3.
 
-## IPWhiteList
+## Static configuration
 
-In v3, we renamed the `IPWhiteList` middleware to `IPAllowList` without changing anything to the configuration. 
+### Docker & Docker Swarm
 
-## gRPC Metrics
+In v3, the provider Docker has been split into 2 providers:
 
-In v3, the reported status code for gRPC requests is now the value of the `Grpc-Status` header.  
+- Docker provider (without Swarm support)
+- Swarm provider  (Swarm support only)
+
+??? example "An example usage of v2 Docker provider with Swarm"
+
+    ```yaml tab="File (YAML)"
+    providers:
+      docker:
+        swarmMode: true
+    ```
+
+    ```toml tab="File (TOML)"
+    [providers.docker]
+        swarmMode=true
+    ```
+
+    ```bash tab="CLI"
+    --providers.docker.swarmMode=true
+    ```
+
+This configuration is now unsupported and would prevent Traefik to start.
+
+#### Remediation
+
+In v3, the `swarmMode` should not be used with the Docker provider, and, to use Swarm, the Swarm provider should be used instead.
+
+??? example "An example usage of the Swarm provider"
+
+    ```yaml tab="File (YAML)"
+    providers:
+      swarm:
+        endpoint: "tcp://127.0.0.1:2377"
+    ```
+
+    ```toml tab="File (TOML)"
+    [providers.swarm]
+        endpoint="tcp://127.0.0.1:2377"
+    ```
+
+    ```bash tab="CLI"
+    --providers.swarm.endpoint=tcp://127.0.0.1:2377
+    ```
+
+### HTTP3 Experimental Configuration
+
+In v3, HTTP/3 is no longer an experimental feature,
+and the associated `experimental.http3` option is now removed in v3.
```

Comment:
```suggestion
In v3, HTTP/3 is no longer an experimental feature.
It can be enabled on entry points without the associated `experimental.http3` option, which is now removed.
```